spec/rspec/rails/fixture_support_spec.rb

```diff
@@ -13,5 +13,54 @@ module RSpec::Rails
         expect(group).to respond_to(:fixture_path=)
       end
     end
+
+    context "without database available" do
+      before { clear_active_record_connection }
+
+      after { establish_active_record_connection }
+
+      context "with use_active_record set to true" do
+        before { RSpec.configuration.use_active_record = true }
+
+        # Not all Rails versions raise the
+        # +ActiveRecord::ConnectionNotEstablished+ exception at this point.
+        # (Eg. Rails 3.x, 4.0, 5.0)
+        it "raise due to no connection established" do
+          example_group = RSpec::Core::ExampleGroup.describe("FixtureSupport") do
+            include FixtureSupport
+            include RSpec::Rails::MinitestLifecycleAdapter
+          end
+
+          test = example_group.example("foo") do
+            expect(true).to be(true)
+          end
+
+          success = example_group.run
+          expect(test.execution_result.exception).to \
+            be_a(ActiveRecord::ConnectionNotEstablished) unless success
```

Comment:
I found this test quite hard to read. Does the `unless success` refers to your comment from line 25 to 27?

If yes, maybe wrap the entire test to the specific Rails version? Like https://github.com/rspec/rspec-rails/blob/85e23f36e1595a3ee37c4eb8271fe459e2030716/spec/rspec/rails/configuration_spec.rb#L259

Also I see more often in RSpec code base
```ruby
expect(
  test.execution_result.exception
).to be_a(ActiveRecord::ConnectionNotEstablished)
```
or
```ruby
 expect(test.execution_result.exception)
   .to be_a(ActiveRecord::ConnectionNotEstablished)
```

instead of 
```ruby
 expect(test.execution_result.exception).to \
            be_a(ActiveRecord::ConnectionNotEstablished)
```